docs/api-docs/partner/support.md

```diff
@@ -8,23 +8,41 @@
 
 ## Developer blog
+
 The [Developer blog](https://medium.com/bigcommerce-developer-blog) provides product, industry, how-to, and community information regularly. Find news, tips, and stories for developing on BigCommerce. 
 
 ## Developer library
+
 The [Developer library](https://developer.bigcommerce.com/dev-library/) provides samples, tools, and guides to help jump-start the perfect ecommerce experience. Find examples and starters to help you begin working on BigCommerce.
 
 ## Developer Slack
-BigCommerce has a dedicated Slack channel for Tech and Agency partners. It is a great place to connect, learn about new updates, and attend “Ask Me Anything” sessions with members of the BigCommerce Product team. To join the BigCommerce developer Slack community, fill out this [form](https://docs.google.com/forms/d/e/1FAIpQLSeQApTpd4T9lEX7hPgUjTyoq0vb8UbiCiWseuurJjdMHGBaGQ/viewform). 
+
+BigCommerce has a dedicated Slack channel for Technology and Agency partners. It is a great place to connect, learn about new updates, and attend “Ask Me Anything” sessions with members of the BigCommerce Product team. To join the BigCommerce developer Slack community, fill out this [form](https://docs.google.com/forms/d/e/1FAIpQLSeQApTpd4T9lEX7hPgUjTyoq0vb8UbiCiWseuurJjdMHGBaGQ/viewform). 
 
 ## Changelog
-[Changelog](https://developer.bigcommerce.com/changelog) is a log of notable updates made to the BigCommerce platform. Find significant changes, such as new features, latest releases, and bug fixes in chronological order.
+
+[Changelog](https://developer.bigcommerce.com/changelog) is a log of notable updates made to the BigCommerce platform. Find significant changes, such as new features, latest releases, and bug fixes in chronological order. To stay up-to-date with the Changelog updates, subscribe to the [RSS Feeds](https://developer.bigcommerce.com/changelog#feeds).
+
+## BigCommerce Status
+
+[BigCommerce Status](https://status.bigcommerce.com/) provides information on the health of BigCommerce-operated services. View [incident history](https://status.bigcommerce.com/history) or subscribe to receive status alerts.
 
 ## Town Hall
+
 [BigCommerce Town Hall](https://support.bigcommerce.com/s/article/BigCommerce-Town-Halls) is a monthly webinar program where merchants and partners receive product and company updates and engage in a live Q&A with BigCommerce executive leadership. Recordings of previous Town Halls are available. 
+
+## Partner Support Team
+
+You can access the Partner Support Team through the Partner Portal. Once you log into your Partner Portal account, hover over the **Support** tab located in the top right corner. Select **Contact Support** and you will be taken to a new page where you can **Open a Support Case** or use our **Live Chat** feature.
```

Comment:
You can access the Partner Support Team through the [Partner Portal](https://partners.bigcommerce.com/). Once you log into your Partner Portal account, hover over the **Support** tab located in the top right corner. Select **Contact Support** and you will be taken to a new page where you can **Create a Support Case** or use our **Live Chat** feature.